### ISSUE TITLE
Installing from pip requires missing package six

### ISSUE THREAD

**Issue by daniel-butler** [amzn/ion-python#196]
Python Version: 3.8.9
amazon.ion Version: 0.9.1

running 
```py
import amazon.ion.simpleion as ion
```

error message:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Volumes/workplace/FaaSTUAMPersonalToolsWorkspace/users_export/.venv/lib/python3.8/site-packages/amazon/ion/simpleion.py", line 28, in <module>
    import six
ModuleNotFoundError: No module named 'six'
```

Proposed fix is to specify six as a required package.

**Comment by cheqianh** [comment]
Thanks for letting us know!  

Since we are using a [required file](https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#requirements-files) for setup, let me see if the file is ignored when ion-python is imported.

**Comment by cheqianh** [comment]
Closed because of fix in #197.
